src/components/Footer.jsx

```diff
@@ -1,13 +1,37 @@
 import React from 'react';
 import styled from 'styled-components';
-import { FOOTER_HEIGHT, COLOR_PALETTE } from '../constants';
+import { /* FOOTER_HEIGHT, */ COLOR_PALETTE } from '../constants';
+
+import PhoneImage from '../images/phone-solid.svg';
+import EmailImage from '../images/envelope-solid.svg';
 
 
 const FooterContainer = styled.footer`
-    height: ${FOOTER_HEIGHT}vh;
+    height: auto;
```

Comment:
The reason I had a fixed height here, is that some pages have content that don't fit the entire height of the page. So what I did was to have the header and footer have a fixed height. Then the actual `main` element have a minimum height of (100vh (full height) minus the height of the footer and header combined). To avoid a situation like this: https://deploy-preview-2--flamboyant-goldwasser-801d4c.netlify.com/students/

For some reason it's also broken in our live website. So it's not a problem now, I'll fix it later. Just thought I'd let you know my original intentions :)